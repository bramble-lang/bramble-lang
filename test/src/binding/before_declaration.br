fn my_main() -> i64 {
    root::std::io::writei64ln(x);
    let x: i64 := 5;
    return;
}