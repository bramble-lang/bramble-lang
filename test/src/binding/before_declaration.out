Error: L2: Variable x not declared
